db/schema.rb

```diff
@@ -52,7 +52,7 @@
   end
 
   create_table "notes", force: :cascade do |t|
-    t.text "body"
+    t.text "body", null: false
```

Comment:
There might be a conflict here from latest master - my fault I missed this from a PR, but Jamie added it.